.github/containers/release/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nvidia/cuda:12.2.2-devel-ubuntu20.04 as base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt -y install curl libssl-dev pkg-config
+RUN curl -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+WORKDIR /src
+
+FROM base AS builder
+COPY . .
+RUN --mount=type=cache,target=/root/.cache \
```

Comment:
Generally its a good idea to include: `,id=<key_name>` other wise the `target=` param will be used as your cache key and you could have cache mixups if running on the same machine. But I think in this case its unlikely.

Unfortantly with the CI machines being short lived, the cache-mounts might not buy a ton of performance since the caches are blown away with the machine existing at end-of-day.

You might consider the s3 layer caching mode we are using in bonsai as a alternative, but it tends to throw out the cache a little too frequently.

Reply:
This was purely a thing for doing local development, it's not intended to work well in AWS at the moment. I might revisit this later once we figure out how to get S3/MemoryDB backend working for caching.